wpeverest/user-registration
# Enhance - Form template publish to remote server

### All Submissions:






### Changes proposed in this Pull Request:
Previously, We are using wp_remote_get to list the form templates from the **plugin directory**. But now we have hosted the form template JSON file on our own server



1. Just changed the URL of the JSON file and images path. 
2. Some typo-related issues fixed

### How to test the changes in this Pull Request:
1. Checkout this branch once and see the form template library and refresh it
2. If everything seems good then you can disable your internet connection for a moment and see the same page
3. Try to import any of the template from available templates

### Types of changes:




### Other information:




### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.
